exams/models.py

```diff
@@ -21,6 +21,8 @@ class ExamRun(TimestampedModel):
     date_first_eligible = models.DateField()
     date_last_eligible = models.DateField()
 
+    date_grades_available = models.DateTimeField()
+
```

Comment:
ehhh... i won't be a stickler right now since the pattern is repeated elsewhere, but i'd prefer no grouping by whitespace for a bunch of small reasons (people having different ideas of 'related', etc). no need to change this now